examples/common_vmseries/main.tf

```
@@ -194,10 +200,13 @@ module "bootstrap" {
 
   for_each = var.bootstrap_storage
 
-  create_storage_account = try(each.value.create_storage, true)
-  name                   = each.value.name
-  resource_group_name    = try(each.value.resource_group_name, local.resource_group.name)
-  location               = var.location
+  create_storage_account           = try(each.value.create_storage, true)
+  name                             = each.value.name
+  resource_group_name              = try(each.value.resource_group_name, local.resource_group.name)
+  location                         = var.location
+  storage_acl                      = try(each.value.storage_acl, false)
+  storage_allow_vnet_subnet_ids    = try(flatten([for v in each.value.storage_allow_vnet_subnet_ids : [module.vnet[v.vnet_key].subnet_ids[v.subnet_key]]]), [])
```

see suggestion below
```suggestion
  storage_allow_vnet_subnet_ids    = try(flatten([for v in each.value.storage_allow_vnet_subnets : [module.vnet[v.vnet_key].subnet_ids[v.subnet_key]]]), [])
```